packages/taquito-michelson-encoder/src/tokens/createToken.ts

```diff
@@ -2,6 +2,10 @@ import { tokens } from './tokens';
 import { Token } from './token';
 import { PairToken } from './pair';
 
+/**
+ *  @category Error
+ *  @description Error that indicates a token having malformed data
```

Comment:
This error can indicate that a script contains an invalid type or a type that is not supported by the Michelson Encoder. 

Maybe we can add a note saying that if the type is valid but not supported, we encourage the user to open an issue and we will add support for it. This can happen if there are new types in a new protocol and we forget to add support for them..